albumentations/augmentations/domain_adaptation.py

```diff
@@ -7,7 +7,10 @@
 from ..core.transforms_interface import ImageOnlyTransform, to_tuple
 from .functional import clipped, preserve_shape
 
-from skimage.exposure import match_histograms
+try:
+    from skimage.exposure import match_histograms
+except ImportError:
+    from skimage.transform import histogram_matching
```

Comment:
Functions has different names

Reply:
Yes you are right it supposed to be from skimage.transform import match_histograms . my mistake.